doc/filters/striptags.rst

```diff
@@ -13,3 +13,11 @@ by one space:
     Internally, Twig uses the PHP `strip_tags`_ function.
 
 .. _`strip_tags`: http://php.net/strip_tags
+
+    As a consequence, a second parameter, `$allowable_tags`, can be provided. In this example, the tags like `<br/>`, `<br>`, `<p>` and `</p>` will remain in the string:
```

Comment:
I would not say ``a second parameter`` but ``a parameter``, as the first argument of a filter does not look like a parameter in the Twig syntax